tests/e2e/assets/golang/go.mod

```diff
@@ -1,3 +1,3 @@
 module dorifi
 
-go 1.20
+go 1.19
```

Comment:
Should we add a comment here? And how do we know when it is safe to bump it?

I am worried Dependabot (or one of us) will bump it back up without realizing.